tapir/accounts/views.py

```diff
@@ -330,3 +333,52 @@ def form_valid(self, form):
             ).save()
 
         return response
+
+
+class MailSettingsView(
+    LoginRequiredMixin,
+    PermissionRequiredMixin,
+    TapirFormMixin,
+    generic.FormView,
+):
+    template_name = "accounts/mailsettings.html"
+    form_class = OptionalMailsForm
+
+    def get_permission_required(self):
+        if self.request.user.pk == self.get_tapir_user().pk:
+            return []
+        return [PERMISSION_ACCOUNTS_MANAGE]
+
+    def get_tapir_user(self) -> TapirUser:
+        return get_object_or_404(TapirUser, pk=self.kwargs["pk"])
+
+    def get_form_kwargs(self):
+        kwargs = super().get_form_kwargs()
+        kwargs.update(
+            {
+                "tapir_user": self.get_tapir_user(),
+            }
+        )
```

Comment:
We can do this in one line I think?
```suggestion
       kwargs["tapir_user"] = self.get_tapir_user()
```